backend_py/primary/primary/routers/parameters/router.py

```diff
@@ -20,10 +19,11 @@ async def get_parameters_and_sensitivities(
     authenticated_user: AuthenticatedUser = Depends(AuthHelper.get_authenticated_user),
     case_uuid: str = Query(description="Sumo case uuid"),
     ensemble_name: str = Query(description="Ensemble name"),
-) -> schemas.EnsembleParametersAndSensitivities:
+) -> Tuple[List[schemas.EnsembleParameter], List[schemas.EnsembleSensitivity]]:
     access = ParameterAccess.from_ensemble_name(authenticated_user.get_sumo_access_token(), case_uuid, ensemble_name)
-    parameters_and_sensitivities = await access.get_parameters_and_sensitivities_async()
-    return schemas.EnsembleParametersAndSensitivities(
-        parameters=converters.to_api_parameters(parameters_and_sensitivities.parameters),
-        sensitivities=converters.to_api_sensitivities(parameters_and_sensitivities.sensitivities),
+    parameters, sensitivities = await access.get_parameters_and_sensitivities_async()
+
+    return (
+        converters.to_api_parameters(parameters),
+        converters.to_api_sensitivities(sensitivities),
     )
```

Comment:
I actually  prefer the schema class `schemas.EnsembleParametersAndSensitivities` we had here for the api return type. I just felt it was unnecessary for the access-layer to wrap it in a new structure and access `parameters_and_sensitivities.parameters` and `parameters_and_sensitivities.sensitivities` immediately afterwards.

i.e. access layer has tuple[list[EnsembleParameter], list[EnsembleSensitivity]] and the router layer here returns 

```
return schemas.EnsembleParametersAndSensitivities(
   parameters=converters.to_api_parameters(parameters),
   sensitivities=converters.to_api_sensitivities(sensitivities),
)
```